addons/assignGear/functions/fnc_getLoadoutFromConfig.sqf

```diff
@@ -72,14 +72,14 @@ _configItems = [_configItems, _containersArray, VEST_INDEX] call FUNC(addItemsTo
 _configItems = [_configItems, _containersArray, BACKPACK_INDEX] call FUNC(addItemsToContainer);
 TRACE_1("Remaining items: ", _configItems);
 
+_configBackpackItems = [_configBackpackItems, _containersArray, UNIFORM_INDEX] call FUNC(addItemsToContainer);
```

Comment:
won't backpack medical go into uniform now instead of backpack?

Reply:
Good call for backpack items